CONTRIBUTING.md

```diff
@@ -1,5 +1,39 @@
 # Contributing
 
+## Templates update
+
+Every team using the template has shared responsibility and code access to this repository.
+The template will be collectively contributed and versioned.
+
+When a bug fix or a feature change is done on the Go template, a pull request on the Java version 
+(preferred) or an issue will be opened to make sure we keep both versions at the same feature and 
+maturity level.
+
+There is an agreement from each fleet-manager team to update their fleet-manager code on the newest 
+template regularly. Code updates to/from the template onto/from a specific service are expected to be 
+done manually since the two repositories are different. You are free to choose/rely on whichever 
+automation tool you have at your disposal e.g the combination of [git patch](https://git-scm.com/docs/git-format-patch) 
+and [git apply](https://git-scm.com/docs/git-apply) etc. Code fix is done on a roll forward fashion.
```

Comment:
Do we expect to change this with phase 2 and phase 3?

Reply:
Nothing is fixed and the guide can be improved on par needs basis as we progress. The current proposal is meant to get things rolling and we'll be revisiting in the future.